oak_functions/sdk/oak_functions/src/lib.rs

```diff
@@ -112,12 +112,10 @@ pub fn report_metric<T: AsRef<str>>(
 ///
 /// These log messages are considered sensitive, so will only be logged by the runtime if the
 /// `oak_unsafe` feature is enabled.
-///
-/// See [`write_log_message`](https://github.com/project-oak/oak/blob/main/docs/oak_functions_abi.md#write_log_message).
 pub fn write_log_message<T: AsRef<str>>(message: T) -> Result<(), OakStatus> {
     let buf = message.as_ref().as_bytes();
-    let status = unsafe { oak_functions_abi::write_log_message(buf.as_ptr(), buf.len()) };
-    result_from_status(status as i32, ())
+    invoke(oak_functions_abi::ExtensionHandle::LoggingHandle, buf)?;
+    Ok(())
```

Comment:
I don't know what the type signature of `invoke()` is, but do we need the explicit `Ok(())` here? If the types match, then just calling `invoke()` without the `?` should work.

Reply:
The return value for invoke is `Result<Vec<u8>>`.